main.go

```diff
@@ -75,10 +81,8 @@ func configureRootCommand() *cobra.Command {
 		false,
 		"if true, the influx client skips https certificate verification")
 
-	_ = cmd.MarkFlagRequired("addr")
+	/* Cannot mark envvar backed arguments as required, must manually check */
```

Comment:
I appreciate the use of comments for context of this PR. However, this comment won't make sense out of context of the PR.
```suggestion
```

Reply:
comment removed in latest commit